.github/workflows/linux.yml

```diff
@@ -165,6 +173,9 @@ jobs:
           timeout_minutes: 20
           max_attempts: 2 # https://github.com/servo/servo/issues/30683
           command: ./mach test-unit --${{ inputs.profile }}
+      - name: Build libservo
+        if: ${{ inputs.build-libservo }}
+        run: cargo build -p libservo
```

Comment:
should we use mach here?

Reply:
this is intentional; mach does not support building examples, and if we want libservo to be consumable as a library, people need to be able to build it without mach.

Reply:
should we also publish the libs?

Reply:
> should we also publish the libs?

answering myself: no because rust abi is not stable.